src/app/frontend/common/components/allocatedresourceschart/component.js

```diff
@@ -63,6 +63,29 @@ export class AllocatedResourcesChartController {
   }
 
   /**
+   * Displays label only for allocated resources (with index equal to 0).
+   *
+   * @return {string}
+   * @private
+   */
+  displayOnlyAllocated(d, i) {
```

Comment:
Private functions should end with `_` suffix.

Reply:
Yep, done.